### PR TITLE
Add some APIs on a JIRA group

### PR DESCRIPTION
**Pull request by b4b4r07**
Added the following endpoints.

- GET `/rest/api/2/group/user`
- POST `/rest/api/2/group/user`
- GET `/rest/api/2/user/groups`

### CODE REVIEW

File: group.go [hunk 1]
```diff
@@ -51,3 +68,50 @@ func (s *GroupService) Get(name string) ([]GroupMember, *Response, error) {
...
+
+	return responseGroup, resp, nil
+}
+
+// Remove removes user from group
+//
+// JIRA API docs: https://docs.atlassian.com/jira/REST/cloud/#api/2/group-removeUserFromGroup
+func (s *GroupService) Remove(groupname string, username string) (*Response, error) {
+	apiEndpoint := fmt.Sprintf("rest/api/2/group/user?groupname=%s&username=%s", groupname, username)
+	// var user struct {
```
Comment (rbriski) [inline group.go]: Unless they're needed as documentation, I'd get rid of the comments.

File: user_test.go [hunk 2]
```diff
@@ -55,3 +55,21 @@ func TestUserService_Create(t *testing.T) {
 		t.Error("Expected user. User is nil")
 	}
 }
+
```
Comment (rbriski) [inline user_test.go]: Can you add tests for the other functions you added as well?  (e.g `Add`, `Remove`)

File: group.go (hunk 1 above)
Comment (b4b4r07) [reply group.go]: Oops, sorry. You're right.

File: user_test.go (hunk 2 above)
Comment (b4b4r07) [reply user_test.go]: Sure

Comment (b4b4r07) [reply user_test.go]: `Add`, `Remove` are APIs of `Group` so I wrote its tests in `group_test.go`.

**Comment by rbriski** [comment]
LGTM

**Comment by andygrunwald** [comment]
Thanks @b4b4r07 for the contribution and @rbriski for the review.

**Pull request merged**